pkg/spec/createconfig.go

```diff
@@ -293,6 +295,39 @@ func (c *CreateConfig) GetVolumeMounts(specMounts []spec.Mount) ([]spec.Mount, e
 	return m, nil
 }
 
+// GetVolumesFrom reads the create-config artifact of the container to get volumes from
+// and adds it to c.Volumes of the curent container.
+func (c *CreateConfig) GetVolumesFrom() error {
+	var options string
+	for _, vol := range c.VolumesFrom {
+		splitVol := strings.Split(vol, ":")
+		if len(splitVol) == 2 {
+			options = splitVol[1]
```

Comment:
Can't there be multiple options?

Reply:
Yeah the format is `ctr:option1,option2` so this just splits it at ":" to get just the options from there, which overrides the default options if given.

Reply:
@umohnani8 Use `strings.SplitN(vol, ":", 2)` then, for slightly better perf and to make it clear there can only be 2 at most